res/lang/missions/missions_en.txt

```diff
@@ -1,37 +1,37 @@
-MISSION               = Task
+MISSION               = Challenge
 MISSION_START_PEP1    = May you do well!
 MISSION_START_PEP2    = Good luck!
 MISSION_START_PEP3    = Do your best!
 MISSION_START_PEP4    = You can do it!
-MISSION_SUCCESS       = %HURRAY%Task %GOAL% finished successfully
+MISSION_SUCCESS       = %HURRAY%Challenge %GOAL% finished successfully
 MISSION_SUCCESS_TEXT1 = Congrationlations!
 MISSION_SUCCESS_TEXT2 = Well done!
 MISSION_SUCCESS_TEXT3 = Great!
-MISSION_END_NEUTRAL1  = What will be your rank in the highscore.
-MISSION_FAILED        = Task %GOAL% failed
-MISSION_OTHER_PLAYER  = %NAME% finished the task first
+MISSION_END_NEUTRAL1  = Let's see where you end up in the high score list.
+MISSION_FAILED        = Challenge %GOAL% failed
+MISSION_OTHER_PLAYER  = %NAME% finished the challenge first
 MISSION_DISABLE_DEV   = (Developer mode was disabled)
-MISSION_ABORTED       = Task aborted
+MISSION_ABORTED       = Challenge aborted
 
-MISSION_CATEGORY_NOGOAL = Endless Game
+MISSION_CATEGORY_NOGOAL = Infinite Game
 MISSION_CATEGORY_IMAGE  = Image
 MISSION_CATEGORY_MONEY  = Money
-MISSION_CATEGORY_REACH  = Reach
+MISSION_CATEGORY_REACH  = Broadcast Area
```

Comment:
To discuss: it's consistent with the translaction of reach in gen_settings. And the area is not quite the same as the number of poeple you reach.

Reply:
I remember "Reach" was unclear in that place in the GUI. Reach what?

Reply:
How about "Audience" for the Title. And "Audience Reach" for the description text ("20% Audience Reach").
I really want to prevent confusion about Area vs Audience.